htdocs/install/mysql/migration/13.0.0-14.0.0.sql

```diff
@@ -79,6 +79,7 @@ DELETE FROM llx_user_param where param = 'MAIN_THEME' and value in ('auguria', '
 ALTER TABLE llx_product_lot ADD COLUMN eol_date datetime NULL;
 ALTER TABLE llx_product_lot ADD COLUMN manufacturing_date datetime NULL;
 ALTER TABLE llx_product_lot ADD COLUMN scrapping_date datetime NULL;
+ALTER TABLE llx_product_lot ADD COLUMN qc_frequency integer DEFAULT NULL;
```

Comment:
This field is inside llx_product

Reply:
Yes and as you said on previous pr, it can be defaulted on product and different on lot